crypto/types.go

```diff
@@ -117,8 +117,17 @@ type Signature []byte
 // invalid inputs.
 type invalidInputsError struct {
 	error
+	subType invalidInputErrorSubType
 }
 
+type invalidInputErrorSubType int
+
+const (
+	Other invalidInputErrorSubType = 0
```

Comment:
`Other` sounds like an exception, but I guess it's not, right? It's just other than `dkgInvalidStateTransition` and `aggregationEmptyList`?

Maybe we should call it `default`. If not specified, it's the default error, otherwise, it could be other specific error like `dkgInvalidStateTransition` or `aggregationEmptyList`.

Reply:
> sounds like an exception, but I guess it's not, right?

Yes

`default` cannot be used because it is a Golang expression, I used `generic` instead, let me know if it's still ambiguous.